draft-ietf-ccwg-bbr.md

```diff
@@ -3127,22 +3129,22 @@ as follows:
     BBR.bdp = BBR.bw * BBR.min_rtt
     return gain * BBR.bdp
 
-  BBRQuantizationBudget(inflight)
+  BBRQuantizationBudget(inflight_cap)
     BBRUpdateOffloadBudget()
-    inflight = max(inflight, BBR.offload_budget)
-    inflight = max(inflight, BBRMinPipeCwnd)
+    inflight_cap = max(inflight_cap, BBR.offload_budget)
+    inflight_cap = max(inflight_cap, BBRMinPipeCwnd)
       if (BBR.state == ProbeBW_UP)
```

Comment:
The indent on this line is a bit odd.  Is this supposed to line up with the two previous lines?

Reply:
Thanks, @ianswett. Good catch! Tried to fix this in a new commit tacked onto this PR:

https://github.com/ietf-wg-ccwg/draft-ietf-ccwg-bbr/pull/41/files#diff-9265ceb047b645a452dd43f3cfa4354bf6d63fb7251f8da997a987cef1758281